ktor-server/ktor-server-plugins/ktor-server-rate-limit/jvmAndNix/src/io/ktor/server/plugins/ratelimit/RateLimit.kt

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.ratelimit
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.util.*
+import io.ktor.util.collections.*
+import io.ktor.util.date.*
+import io.ktor.util.logging.*
+import kotlin.time.*
+
+internal val LOGGER = KtorSimpleLogger("io.ktor.server.plugins.RateLimit")
```

Comment:
for consistency with other plugins - `io.ktor.server.plugins.ratelimit.RateLimit`
(BTW, not all plugins are correct in this, f.e. [Routing](https://github.com/ktorio/ktor/blob/7384555da1c077aab9e812a368ba497a3437bb6f/ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/routing/Routing.kt#L19) and [OAuth](https://github.com/ktorio/ktor/blob/7384555da1c077aab9e812a368ba497a3437bb6f/ktor-server/ktor-server-plugins/ktor-server-auth/jvm/src/io/ktor/server/auth/OAuth.kt#L18))